templates/packages.erb

```diff
@@ -13,20 +10,14 @@ httpd-itk
 ipset
 iptables
 kernel
-mariadb
-mariadb-server
 memcached
-mg
 nano
 net-tools
-nginx
 ntp
 ntpdate
 openssh
 openssh-server
 openssl
-php
```

Comment:
This, and others, will be needed for the classroom training labs. Are the list of packages for the training/puppetfactory VM elsewhere?

Reply:
Nevermind, I see the unless.